docs/policies.md

```diff
@@ -66,17 +66,50 @@ privileges [here](https://github.com/datahub-project/datahub/blob/master/metadat
 
 All edits on the UI are covered by a privilege, to make sure we have the ability to restrict write access.
 
-<!---
-TODO: Add table for edit privileges
---->
+We currently support the following:
+
+**Platform-level** privileges for DataHub operators to access & manage the administrative functionality of the system.
+
+| Platform Privileges             | Description                                                                                                                    |
+|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| MANAGE_POLICIES                 | Allow actor to create and remove access control policies. Be careful - Actors with this privilege are effectively super users. |
```

Comment:
Each privilege has a "display name" which is what people using the UI actually see. Maybe we can have the first column contain the display name, and have the internal type somewhere less prominent?

Overall looks great!